clang/lib/Format/ContinuationIndenter.cpp

```diff
@@ -304,6 +304,10 @@ bool ContinuationIndenter::canBreak(const LineState &State) {
                                    Current.closesBlockOrBlockTypeList(Style))) {
     return false;
   }
+  if (Style.BreakBeforeTemplateCloser && Current.is(TT_TemplateCloser) &&
+      !CurrentState.BreakBeforeClosingAngle) {
+    return false;
+  }
```

Comment:
```suggestion
  if (Style.BreakBeforeTemplateCloser && Current.is(TT_TemplateCloser))
    return CurrentState.BreakBeforeClosingAngle;
```
And move this to line 362 below for ease of maintenance.